### DISCUSSION TITLE
Sending data from CDS back to home assistant

### DISCUSSION THREAD

**Discussion by plugfir3** [BimmerGestalt/HassGestalt#18] [Ideas]
Thanks for the great app! I really enjoy using it.

Are there alternative methods for transmitting data, such as fuel level information from the CDS, back to home assistant, in addition to controlling entities within the system?

**Comment by hufman** [comment]
HassGestalt does not support transmitting data from the car into Home Assistant. I use the normal BMW Connected Drive integration, which provides the data relevant to my automations.

**Reply by plugfir3** [reply]
Sadly, my car does not support connected drive, but EntryNav. Thanks anyway. Really appreciated your work.
